.github/workflows/lint.yml

```diff
@@ -13,7 +13,10 @@ jobs:
     steps:
       - name: Checkout repository
         uses: actions/checkout@v3
-      - name: Run ShellCheck
+      # This workflow gets injected into other Linux repositories, but we don't
```

Comment:
Can you move the condition  just before the actual steps (line 13), I believe this will make the whole job a no-op vs performing some of the steps.

Reply:
Sure thing, done.